docs/compose/Modal/README.md

```diff
@@ -0,0 +1,104 @@
+
+## ModalWithoutAction
+
+| Day | Night | Right to Left |
+| --- | --- | --- |
+| <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/Modal/screenshots/modal-title-no-action.png" alt="ModalWithoutAction" width="375" /> | <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/Modal/screenshots/modal-title-no-action_dm.png" alt="ModalWithoutAction - dark mode" width="375" /> | <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/Modal/screenshots/modal-title-no-action_rtl.png" alt="ModalWithoutAction - rtl mode" width="375" /> |
+
+## ModalWithoutActionAndTitle
+
+| Day | Night | Right to Left |
+| --- | --- | --- |
+| <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/Modal/screenshots/modal-no-title-no-action.png" alt="ModalWithoutActionAndTitle" width="375" /> | <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/Modal/screenshots/modal-no-title-no-action_dm.png" alt="ModalWithoutActionAndTitle - dark mode" width="375" /> | <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/Modal/screenshots/modal-no-title-no-action_rtl.png" alt="ModalWithoutActionAndTitle - rtl mode" width="375" /> |
+
+## ModalWithBackIcon
+
+| Day | Night | Right to Left |
+| --- | --- | --- |
+| <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/Modal/screenshots/modal-back-icon.png" alt="ModalWithBackIcon" width="375" /> | <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/Modal/screenshots/modal-back-icon_dm.png" alt="ModalWithBackIcon - dark mode" width="375" /> | <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/Modal/screenshots/modal-back-icon_rtl.png" alt="ModalWithBackIcon - rtl mode" width="375" /> |
+
+## Installation
+
+Backpack Compose is available through [Maven Central](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-compose). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
+
+## Usage
+
+Example of a Modal with title, text action and text content:
+
+```Kotlin
+import net.skyscanner.backpack.compose.net.skyscanner.backpack.compose.dialog.BpkModal
+
+BpkModal(
+    title = stringResource(id = R.string.dialog_title),
+    closeButtonAccessibilityLabel = stringResource(id = R.string.navigation_accessibility),
+    action = TextAction(
+        text = stringResource(R.string.navigation_text_action),
+        onClick = { /** onActionClick **/ },
+    ),
+    onDismiss = { /** onDismiss **/ },
+) {
+    BpkText(
```

Comment:
you can just specify this as ` // content` rather than putting fake content here